.eslintrc.js

```diff
@@ -8,9 +8,7 @@ module.exports = {
     'prefer-arrow-callback': ['error', {
       allowNamedFunctions: true,
     }],
-    'import/extensions': ['error', 'always', {
-      ignorePackages: true,
-    }],
+    'import/extensions': 'off',
```

Comment:
We might still want this. One of the common "slowness" of webpack is resolving file paths to actual files. If it has to guess which extension it is, it slows it down a bit at a time.